### PR TITLE
fix: disable comments on Jest testing

### LINKED ISSUE #50: Disable Coverage Report Comment
I don't like the notifications on every commit. Please do not post these as a comment. 

https://github.com/ubiquity/webauthn-evm-signing-key/commit/59301faa885d775c50de3e80d200b947717a2b8b#commitcomment-143320074

https://github.com/ubiquity/ts-template/blob/development/.github/workflows/jest-testing.yml#L25-L31

### PR DESCRIPTION
**Pull request by gentlementlegen**
Resolves #50

### CODE REVIEW

**Comment by gentlementlegen** [comment]
No more comment is posted, only the result can be visible within the Action run itself: https://github.com/ubiquity/ts-template/actions/runs/9617643273/job/26529830238?pr=51#step:4:41

If that is satisfactory, I will also apply this within https://github.com/ubiquibot/plugin-template

---

I do not know if that is interesting to have but we could also add the summary on the Action run like so: https://github.com/Meniole/ts-template/actions/runs/9617961749/attempts/1#summary-26530846974

**Comment by 0x4007** [comment]
Summary on the action seems like a good solution. Just as long as you verify that it doesn't produce a notification.

**Review by 0x4007 — APPROVED** [review]

**Comment by gentlementlegen** [comment]
@0x4007 I should hopefully not. If it ever does, I will remove [this section](https://github.com/ubiquity/ts-template/pull/51/files#diff-1679925090c8e7b74c559c158d99ce8d33a73576d608ddfeea14007062208d04R25) right away. But afaik, since it's just tied to the Action run itself, it should not.

[This run](https://github.com/ubiquity/ts-template/actions/runs/9621178410) has the summary linked to itself.

File: cypress/e2e/main.cy.ts [hunk 1]
```diff
@@ -7,5 +7,6 @@ describe("Homepage tests", () => {
     });
     cy.get("@consoleError").should("not.be.called");
     cy.get("body").should("exist");
+    cy.get("h1").should("exist");
```
Comment (0x4007) [inline cypress/e2e/main.cy.ts:10]: Why was this added?

Comment (gentlementlegen) [reply cypress/e2e/main.cy.ts:10]: Not very relevant to this PR, just adding more coverage while I was testing, so I thought it could remain there.
